lib/CallBackery/GuiPlugin/Users.pm

```diff
@@ -127,6 +129,7 @@ has actionCfg => sub {
             }
         },
         $admin ? ({
+            key => 'deluser',
```

Comment:
key already exists on line 136

Reply:
Ah, removed and renamed the other two to be more consistent.